modules/vcf2maf/1.1/vcf2maf.smk

```diff
@@ -76,6 +77,7 @@ rule _vcf2maf_run:
         mem_mb = CFG["mem_mb"]["vcf2maf"]
     shell:
         op.as_one_line("""
+        if [[ -e {output.maf} ]]; then rm -f {output.maf}; fi;
```

Comment:
Should we add this same logic for the intermediate vep file? If it is not deleted by the rule then it actually prevents VEP from re-running. VCF2maf just assumes that it's been run and uses the old vep.vcf as input.